Munkanapló frissítése

BaranyKristof opened the pull request:
Frissítettem a munkanaplót az általam eddig elvégzett munka szerint.

lucakovacs committed:
Maven beüzemeléshez tartozó dokumentáció #6

lucakovacs committed:
CI beüzemelés dokumentációja #2 #6

lucakovacs committed:
Munkanapló feltöltése #6

BaranyKristof committed:
Munkanapló feltöltése #6

Munkanapló a jelenlegi állapotában

lucakovacs commented on doc/IET munkanapló sablon.docx:
@BaranyKristof a 3. részfeladathoz nem tudom, hogy írni szerettél volna még, vagy sem. Ha igen akkor pótold, ha nem akkor vedd ki belőle azt a részt:
![image](https://github.com/BME-MIT-IET/iet-hf-2023-csapat1000/assets/22234422/d6e99f22-6261-409e-8075-e65f2a4b1000)

jud11t committed:
Kódlefedettség dokumentáció #20 #6

jud11t committed:
IET munkanapló frissítése #6

jud11t committed:
IET munkanapló sablon.docx frissítése #6

Katinka666 committed:
Munkanapló frissítése

Katinka666 committed:
Munkanapló frissítése #6

lucakovacs committed:
Apró módosítás a munkanaplóban #6

Katinka666 committed:
Dokumentáció elkészítése a manuális teszteléshez #6

Megírtam és a projekthez adtam a dokumentációt, melyben a maunális tesztelés során elvégzett munkám folyamatát rögzítettem.

Katinka666 committed:
Merge branch 'issue06-doc' of https://github.com/BME-MIT-IET/iet-hf-2023-csapat1000 into issue06-doc

jud11t committed:
JUnit doksi #5 #6

(cherry picked from commit d160d0afd4cfa7af430bd63d24501a27f3ab79f2)

BaranyKristof committed:
Munkanapló frissítése #6

BaranyKristof committed:
Ideiglenes munkanapló törlése

BaranyKristof committed:
Stresszteszt dokumentáció

BaranyKristof committed:
Stressztesztelés dokumentáció #16 #6

kololilla committed:
Statikus analízis dokumentáció hozzáadva #6

kololilla committed:
Munkanapló kiegészítés #6 #18 #24
